### PR TITLE
v1.6.0-beta.1 release

### PR DESCRIPTION
**Pull request by ketankelkar**
##### SUMMARY

Contains all changes for 1.6.0-beta.1 release including updated docs (.rst files), meta files, changelogs, release notes and of course updated module src code. Addresses issue #606 

##### ISSUE TYPE
- New Beta Release PR

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -1,17 +1,63 @@
...
-.. © Copyright IBM Corporation 2020, 2021                                          .
+.. © Copyright IBM Corporation 2020, 2021, 2023                              .
 .. ...........................................................................
 
 ========
 Releases
 ========
-
-Version 1.5.0-beta.1
+Version 1.6.0-beta.1
```
Comment (ddimatos) [inline docs/source/release_notes.rst:8]: Missing space after "====" and then remove the other space after 1.6.0-beta.1

File: docs/source/release_notes.rst [hunk 2]
```diff
@@ -1,17 +1,63 @@
...
 
+- ``zos_volume_init`` - Can initialize volumes or minidisks on target z/OS systems which includes creating a volume label and an entry into the volume table of contents (VTOC).
+
+Minor Changes
+-------------
+
+- ``zos_blockinfile`` - Adds an enhancement to allow double quotes within a block.
+- ``zos_copy`` - Adds an enhancement to keep track of modified members in a destination dataset, restoring them to their previous state in case of a failure.
+- ``zos_data_set`` - Adds a new option named force to enable deletion of a data member in a PDSE that is simultaneously in use by others.
+- ``zos_job_query`` - Enables embedded positional wild card placement throughout `job_name` and `job_id` parameters.
```
Comment (ddimatos) [inline docs/source/release_notes.rst:22]: I don't think RST does anything with single single  back ticks "`" , if you double then `` you will get colorized text might be nice for the module properties, we did this in 1.4.0 doc as an example. Some other options here in the release notes might benefit.

**Review by ddimatos — CHANGES_REQUESTED** [review]
Looks real good @ketankelkar nice job. I might suggest that @rexemin take a look at the zos_copy auto merge changes, overall everything I see here looks to be as it should

**Review by rexemin — APPROVED** [review]
The changes to `zos_copy` look good to me.

**Comment by ddimatos** [comment]
Full regression log `/job/zosAnsibleAlternate/54/consoleText`
image
Nice job, over 1000 test cases 👍

**Review by ddimatos — APPROVED** [review]
Looks good, thanks a bunch @ketankelkar

**Pull request merged**